vendors/patterns.yml

```diff
@@ -41,4 +41,42 @@ patterns:
       - name: sendgrid.py
         start_offset: 20
         end_offset: 89
+
+  - name: Okta token
+    type: okta_token
+    regex:
+      version: 0.1
+      pattern: |
+        (0{2}[0-9A-Za-z\-\_]{40})
+      start: |
+        (\A|[^0-9A-Za-z\_\-0x])
+      end: |
+        (\z|[^0-9A-Za-z_=-])
+      additional_not_match:
+        # to cut stuff where there is just numbers
+        - (\d\D|\D\d)
+        # Positive match
+        - [g-zG-Z]
+      additional_match:
+        # Avoid Hexadecimal only 
+        - (^[0-9A-Fa-f\-]+$)
```

Comment:
lose the braces around the string - so just `^[0-9A-Fa-f\-]+$`. Just readability really

Reply:
I suggest adding these optional `0x` prefix and `x` trailing parts to this, since I saw those in my searches over GH:

```
^(0x)?[0-9A-Fa-f\-]+x?$
```